oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -328,6 +318,47 @@ void CodeUpdate::setVersions()
                         "xyz.openbmc_project.Software.Activation";
                     auto imageObjPath = path.str.c_str();
 
+                    // If the version interface is added with the Activation
+                    // value as Invalid, it is considered as assemble code
+                    // update image failure Failure is occured during assembling
+                    // lid files are creating a tar ball.
+
+                    try
+                    {
+                        auto propVal =
+                            pldm::utils::DBusHandler().getDbusPropertyVariant(
+                                imageObjPath, "Activation", imageInterface);
+                        const auto& activation = std::get<std::string>(propVal);
+
+                        if (activation ==
+                            "xyz.openbmc_project.Software.Activation.Activations.Invalid")
+                        {
+                            std::cerr << "Received Invalid Signal, Sending "
```

Comment:
This msg may make more sense to move it inside the "if (isCodeUpdateInProgress())" check because if inband code update is not in progress then the notification won't be sent to phyp.

Reply:
done